app.rb

```diff
@@ -1,12 +1,111 @@
+require_relative 'genre'
+require_relative 'music_album'
+require 'json'
+
 class App
+  def initialize
+    @music_albums = []
+    @genres = []
+  end
+
+  def load_data
+    load_music_albums_data
+    load_genres_data
+  end
+
+  def load_file(file_path)
+    return [] unless File.exist?(file_path) && File.size(file_path).positive?
+
+    JSON.parse(File.read(file_path))
```

Comment:
I think it would make sense to leverage Ruby's JSON class, as it has a `load_file` method that runs this exact line. More info [here](https://ruby-doc.org/stdlib-3.0.2/libdoc/json/rdoc/JSON.html#method-i-load_file:~:text=load_file(path%2C%20opts%3D%7B%7D)%20%E2%86%92%20object).